modules/sending-training-data-to-models.adoc

```diff
@@ -0,0 +1,38 @@
+:_module-type: PROCEDURE
+
+[id="sending-training-data-to-models_{context}"]
+= Sending training data to models
```

Comment:
should this be plural or singular - maybe "Sending training data to a model"?

Reply:
need to be consistent throughout the module (prereqs use "model" while title and step 5 use "models"

Reply:
I really think this should be singular model rather than models to be consistent with the titles of other modules, for example: Viewing a deployed model, Configuring bias metrics for a model

Line 7 in this file now uses singular so if you strongly disagree and think that this should be plural, then change line 7 to be consistent

thx